### DISCUSSION TITLE
[Icon][Windows] Finale Poll

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1496] [Polls]
The Finale will ends the 1st May at midnight.

The taskbar will use the icon of a similar size as of the small one, click on the image to see it at full size.

| icon | preview |
|-|-|
| `circle` |![Screen Shot 2023-04-19 at 12 55 27 AM](https://user-images.githubusercontent.com/587742/232922443-455b90a6-6cdc-4df3-bd95-e1b3c0219618.png)| 
| `nobg` | ![Screen Shot 2023-04-19 at 12 55 45 AM](https://user-images.githubusercontent.com/587742/232922497-90548333-5f39-4ead-9259-cdf5ea51dee8.png) |

**Comment by eduarddejong** [comment]
Somewhat difficult choice, but the nobg version looks a bit more consistent with the normal VS Code logo to me.
The white circle version is very visible, but it does not change with dark or light theme. The nobg version is more subtile, which I think is a good thing.

**Comment by daiyam** [comment]
Winner: `nobg`(57%)
